packages/hs-react-ui/src/components/Table/Table.tsx

```diff
@@ -1,48 +1,83 @@
 import React, { useState, useEffect } from 'react';
 import styled, { StyledComponentBase } from 'styled-components';
 import useResizeObserver from 'use-resize-observer';
+import { mdiArrowDown, mdiChevronRight, mdiChevronDown, mdiChevronUp } from '@mdi/js';
 import Icon from '@mdi/react';
-import { mdiArrowDown } from '@mdi/js';
-
 import colors from '../../enums/colors';
 import fonts from '../../enums/fonts';
 
+/** Types and Interfaces */
+
+export type CellOptions = {
+  RenderedCell: any;
+  headerColumnKey: string; //
+  breakPointHit: boolean;
+  row: columnTypes;
+  index: number;
+  indexModifier?: number;
+  CollapseIcon?: any;
+  groupIndex?: number;
+  isCollapsed?: boolean;
+  cIcon?: any;
+  eIcon?: any;
+  groupLabelDataString?: string;
+};
+
+export type ExpansionIconProps = {
+  collapsedIcon: string;
+  expandedIcon: string;
```

Comment:
should these be `any`?

Reply:
These are for the internal `ExpansionIcon` component that shows the arrow down/right by default. The developer has the ability to either pass in a string path to override the default without anything other than passing in a string of the path for a material icon - or if they choose they can completely ignore our default and pass their own component to be rendered. This type also shouldn't be exported as it's internal only and not user-facing.